src/middleware/codeSplit.js

```diff
@@ -1,11 +1,16 @@
 import { createStore, applyMiddleware, compose, combineReducers } from 'redux'
 
+import { tryRequire } from '../AsyncComponent'
 
 export default (name = 'load') => (api) => async (req, next) => {
   const load = req.route && req.route[name]
 
   if (load) { // if `route.load` does not exist short-circuit
-    const parts = await load(req)
+    let parts = await load(req)
+    if (typeof parts === 'number') {
+      parts = tryRequire(parts)
```

Comment:
I'm an asshole. We've been so close all this time. Good work. Im gonna be putting this to work in actual project now. I wonder if we need to use `tryRequire` to get access to the parts. The point is `load` was supposed to provide them...i'll figure it out.

Reply:
Hope you're alright mate!

Reply:
All good bro, just nice to have you back in the swing of things! LMK how we can be of service.

Reply:
@hedgepigdaniel whats good brotha, we never got to fully connect, but we will soon. hope all is treating u good down under. once i get the docs out, u guys will know its on.